### PR TITLE
Wiswedel/inherited shares/design polishing

### LINKED ISSUE #18202: Inherited shares design review
Design review :)
- [ ] Main row text should be either only "Others with access", or "4 others with access" with the number prepended: __decision is to never show the number__
- [ ] Agree with @rullzer, let’s use "Added by" instead of "Invited by", better language
- [ ] "Open folder" is a bit confusing as it’s unclear what it does. It opens the parent folder which this person got shared. Can we just write the folder name here, as per the mockup? If that’s not possible, we should write "Via parent folder".

For another pr as we should change all the other sharing entries using this wording & icon
- [ ] ~~"Delete share" sounds like it deletes some content. Can it be "Unshare"~~
- [ ] ~~Also, can it have the icon-close as icon instead of the delete icon?~~

Answering your questions @skjnldsv 

>     Should the "open folder" action open in a newt tab?

Yes, probably.

>     Should we display inherited shares that are exactly the opened folder ? (e.g I can currently see shares of the folder 123 when opening the list of inherited shares on the folder 123)

No? Cause they are just shares, not inherited shares? Or do I misunderstand?

_Originally posted by @jancborchardt in https://github.com/nextcloud/server/pull/15952#issuecomment-559504090_

### PR DESCRIPTION
**Pull request by wiswedel**
resolves https://github.com/nextcloud/server/issues/18202

Signed-off-by: Sascha Wiswedel <sascha.wiswedel@nextcloud.com>

### CODE REVIEW

**Review by skjnldsv — APPROVED** [review]
You rock!

**Comment by jancborchardt** [comment]
@wiswedel cool, thanks! Could you add screenshots for quicker design review? Thanks! :)

**Comment by wiswedel** [comment]
> Could you add screenshots for quicker design review? Thanks! :)

image

@jancborchardt

File: apps/files_sharing/src/components/SharingEntryInherited.vue [hunk 1]
```diff
@@ -33,17 +33,17 @@
...
 		</template>
 		<ActionText icon="icon-user">
-			{{ t('files_sharing', 'Invited by {initiator}', { initiator: share.ownerDisplayName }) }}
+			{{ t('files_sharing', 'Added by {initiator}', { initiator: share.ownerDisplayName }) }}
 		</ActionText>
 		<ActionLink v-if="share.fileSource"
 			icon="icon-folder"
 			:href="fileTargetUrl">
-			{{ t('files_sharing', 'Open folder') }}
+			{{ t('files_sharing', 'Via folder') }}
```
Comment (jancborchardt) [inline apps/files_sharing/src/components/SharingEntryInherited.vue]: "Via parent folder" would be clearer, as per https://github.com/nextcloud/server/issues/18202

**Comment by jancborchardt** [comment]
Looking nice! See the comment on the wording, and if we also get "icon-close" for the Unshare action then it’s as per spec in the issue and good to go. :)

File: apps/files_sharing/src/components/SharingEntryInherited.vue (hunk 1 above)
Comment (wiswedel) [reply apps/files_sharing/src/components/SharingEntryInherited.vue]: I don't really like either.
- "Parent folder" implies _parent of the one I'm currently in_ to me.
- "Via folder" is also kind of bumpy wording.

@jancborchardt Do you have any other smart ideas?

**Comment by wiswedel** [comment]
> Looking nice! See the comment on the wording, and if we also get "icon-close" for the Unshare action then it’s as per spec in the issue and good to go. :)

icon-close contradicts with the normal unshare icon

image

**Comment by skjnldsv** [comment]
> Looking nice! See the comment on the wording, and if we also get "icon-close" for the Unshare action then it’s as per spec in the issue and good to go. :)

Yes, we said this should be done in a dedicated pr

**Comment by wiswedel** [comment]
> > Looking nice! See the comment on the wording, and if we also get "icon-close" for the Unshare action then it’s as per spec in the issue and good to go. :)
> 
> Yes, we said this should be done in a dedicated pr

Yeah, let's do both then in a separate PR, but not have inconsistencies in the meantime.

File: apps/files_sharing/src/components/SharingEntryInherited.vue (hunk 1 above)
Comment (jancborchardt) [reply apps/files_sharing/src/components/SharingEntryInherited.vue]: Then say "Via `foldername`", actually naming the folder in question. :)

If that is not possible, "Via parent folder" is the best compromise between concise and informative atm. We can always enhance this later.

Comment (wiswedel) [reply apps/files_sharing/src/components/SharingEntryInherited.vue]: > Via `foldername`
I like that - however, that's where my vue half knowledge ends. Can you help me retrieving that folder name @jancborchardt ?

Comment (jancborchardt) [reply apps/files_sharing/src/components/SharingEntryInherited.vue]: No, I likely have even less knowledge there. :) @skjnldsv @juliushaertl?

**Review by ChristophWurst — APPROVED** [review]

**Pull request merged**